cli/command/system/info.go

```diff
@@ -373,9 +366,8 @@ func formatInfo(dockerCli *command.DockerCli, info types.Info, format string) er
 	return err
 }
 
-func fprintfIfNotEmpty(w io.Writer, format, value string) (int, error) {
+func fprintNonEmpty(w io.Writer, label, value string) {
```

Comment:
This should be `fprintlnNonEmpty` to make it clear that this is backed by `fmt.Fprintln`, which has slightly different behavior for spacing from `fmt.Fprint`.